Tasks/AzureTestPlanV0/Invokers/maveninvoker.ts

```diff
@@ -33,10 +33,12 @@ export async function executeMavenTests(testsToBeExecuted: string[], pomFilePath
         args.push(pomFilePath);
     }
 
+    args.push('-Dmaven.test.failure.ignore=true');
```

Comment:
Nit: Add a comment above this line with purpose of this argument and documentation link of this argument.

Reply:
added comment and documentation link